<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PresenterUIClass</class>
 <widget class="QMainWindow" name="PresenterUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PresenterUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Questions:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="questionListWidget"/>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Id</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="idLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Text</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="textLineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Answer</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="answerLineEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Score</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="scoreLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="questionPushButton">
      <property name="text">
       <string>ADD QUESTION</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>questionPushButton</sender>
   <signal>clicked()</signal>
   <receiver>PresenterUIClass</receiver>
   <slot>addQuestion()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>811</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addQuestion()</slot>
 </slots>
</ui>
